### PR TITLE
Fix forum

### PR DESCRIPTION
**Pull request by phibel**

### CODE REVIEW

**Comment by CaMer0n** [comment]
Excellent @phibel ! Thank you!
I have no time to test, but maybe @rica-carv or @Jimmi08 would like to. 

The code looks good to me though. 
Merging.

**Comment by phibel** [comment]
Thank you @CaMer0n, did you read my commit message 39f42fda6ba21bb467cb433c018176e67e11dfff ? Any thoughts about that?

File: e107_plugins/forum/forum_class.php [hunk 1]
```diff
@@ -2527,27 +2527,24 @@ function threadDelete($threadId, $updateForumLastpost = true)
...
 		$sql 		= e107::getDb();
 		$deleted 	= false;
 
 		$postInfo   = $sql->retrieve('forum_post', '*', 'post_id = '.$postId);
-		//if(!$sql->select('forum_post', '*', 'post_id = '.$postId))
+
 		if(!is_array($postInfo) || empty($postInfo))
 		{
-			echo 'NOT FOUND!'; return;
+			return null;
```
Comment (phibel) [inline e107_plugins/forum/forum_class.php]: Changed the return here to "null" and adapted the description.

Comment (CaMer0n) [reply e107_plugins/forum/forum_class.php]: :+1:

**Pull request merged**